modules/new_profile/php/NDB_Form_new_profile.class.inc

```diff
@@ -297,6 +297,16 @@ class NDB_Form_New_Profile extends NDB_Form
                     $errors['PSCID'] = 'PSCID has already been registered';
             }
         }
+
+        // validate site entered
+        $user = \User::singleton();
+        $user_list_of_sites = $user->getData('CenterIDs');
+        $site = $values['psc'];
+
+        if (empty($site) || !in_array($site, $user_list_of_sites, true)) {
+            $errors['psc'] = "Site must be selected from the available dropdown.";
+        }
+
```

Comment:
why not just use $user->hasCenter()?

Reply:
thats a thing ?

Reply:
@driusan I will need to modify the hasCenter function